_docs/style/text-conventions/date-time.md

```diff
@@ -1,26 +1,32 @@
 ---
-title: Date & Time
+title: Date and time
```

Comment:
we need to identify/establish Helix standard on localization of date and time. See the content Catherine posted in the # docs  channel